tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -49,6 +48,53 @@ bool DebuggerProbesInstrumentationRequester::IsCoreLibOr3rdParty(const WSTRING&
     return false;
 }
 
+WSTRING DebuggerProbesInstrumentationRequester::GenerateRandomProbeId()
+{
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    std::uniform_int_distribution<> dis(0, 15);
+    std::uniform_int_distribution<> dis8(8, 11);
+
+    std::wstringstream ss;
```

Comment:
we have a `shared::WSTRINGSTREAM`, you could use instead of a `std::wstringstream` and converting the result string into a `WSTRING`

Reply:
We will need an alternative for this due to: https://github.com/DataDog/dd-trace-dotnet/blob/2e6bd56d8d2339c62239d1c98e98f0bee00faaa5/shared/src/Datadog.Trace.ClrProfiler.Native/instrumented_assembly_generator/instrumented_assembly_generator_helper.h#L323-L324

Reply:
Yep, I think I'll keep it as is for now. Resolving.